tests/test_experiment.py

```diff
@@ -611,3 +620,128 @@ def test_experiment_stop_does_not_raise_on_unknown_job_id(
     assert stat == InvalidJobStatus.NEVER_STARTED
     after_cancel = exp.get_status(*all_known_ids)
     assert before_cancel == after_cancel
+
+
+def test_start_sequences_of_jobs_good(test_dir, wlmutils, monkeypatch, job_list):
+    """Test the unpacking of a tuple of tuples
+    exp.start(job1, (job2, job3))
+    """
+    monkeypatch.setattr(
+        "smartsim._core.dispatch._LauncherAdapter.start",
+        lambda launch, exe, job_execution_path, env, out, err: random_id(),
+    )
+
+    ensemble = Ensemble("ensemble-name", "echo", replicas=2)
+
+    launch_settings = launch_settings.LaunchSettings(wlmutils.get_test_launcher())
+    job_list = ensemble.build_jobs(launch_settings)
+
+    exp = Experiment(name="exp_name", exp_path=test_dir)
+
+    exp.start(job_list)
```

Comment:
Is this test supposed to be parameterized? It looks like it's requesting a `job_list` fixture, but is unable to find one.

If not, feel free to remove `job_list` from the fn signature.


If so, just a heads up, line 637 is overwriting the value before it is used. You may want to drop that line!